kernel/univ.mli

```diff
@@ -453,10 +461,10 @@ val abstract_cumulativity_info : cumulativity_info -> universe_level_subst * abs
 
 val make_abstract_instance : abstract_universe_context -> universe_instance
 
-(** Get the instantiated graph. *)
+(** Don't use. *)
```

Comment:
This exposes the name of bound variables, and as such it breaks the abstraction. All uses should be eventually eradicated, in favour of `AUContext.instantiate`.